docs/guides/migrating-from-camunda-platform-7/adjusting-bpmn-models.md

```diff
@@ -0,0 +1,205 @@
+
+- `camunda:topic` becomes `zeebe:taskDefinition type`.
+
+The following attributes/elements cannot be migrated:
+
+- `camunda:taskPriority`
+
+Service tasks using `camunda:connector` will be migrated with the following changes:
+
+- `camunda:connectorId` becomes `zeebe:taskDefinition type`
+- All inputs and outputs are treated like all other inputs and outputs.
+
+### Send tasks
+
+![Send Task](../../components/modeler/bpmn/assets/bpmn-symbols/send-task.svg)
+
+In both engines, a send task has the same behavior as a service task. A send task is migrated exactly like a service task.
+
+### Gateways
+
+Gateways rarely need migration. The relevant configuration is mostly in the [expressions](/components/concepts/expressions.md) on outgoing sequence flows.
+
+### Expressions
+
+Expressions need to be in [FEEL (friendly-enough expression language)](/components/concepts/expressions.md#the-expression-language) instead of [JUEL (Java unified expression language)](https://docs.camunda.org/manual/latest/user-guide/process-engine/expression-language/).
+
+Migrating simple expressions is doable (as you can see in [these test cases](https://github.com/camunda-community-hub/camunda-7-to-8-migration/blob/main/backend-diagram-converter/core/src/test/java/org/camunda/community/migration/converter/ExpressionTransformerTest.java)), but not all expressions can be automatically converted.
+
+The following is not possible:
+
+- Calling out to functional Java code using beans in expressions
+- Registering custom function definitions within the expression engine
+- using SPIN library (the FEEL data structure behaves like JSON natively, so SPIN can be omitted or replaced)
+- using the `execution` or `task`
+
+### Human tasks
+
+![User Task](../../components/modeler/bpmn/assets/bpmn-symbols/user-task.svg)
+
+Human task management is also available in Camunda Platform 8, but uses a different tasklist user interface and API.
+
+In Camunda Platform 7, you have [different ways to provide forms for user tasks](https://docs.camunda.org/manual/latest/user-guide/task-forms/):
+
+- Embedded Task Forms (embedded custom HTML and JavaScript)
+- External Task Forms (link to custom applications)
+- [Camunda Forms](/guides/utilizing-forms.md)
+
+Only Camunda Forms are currently supported in Camunda Platform 8 and can be migrated.
+
+The following attributes/elements can be migrated:
+
+- Task assignment (to users or groups):
+  - `bpmn:humanPerformer`
+  - `bpmn:potentialOwner`
+  - `camunda:assignee`
+  - `camunda:candidateGroups`
+  - `camunda:formKey`, but Camunda Platform 8 requires you to embed the form definition itself into the root element of your BPMN XML models, see [this guide](/guides/utilizing-forms.md#connect-your-form-to-a-bpmn-diagram).
+
+The following attributes/elements cannot (yet) be migrated:
+
+- `camunda:candidateUsers` (only candidate groups are supported)
```

Comment:
This has changed in 8.2, candidateUsers are supported now.

Reply:
I removed "(only candidate groups are supported)" - [8a54ad9](https://github.com/camunda/camunda-platform-docs/pull/2283/commits/8a54ad9cf01139a975a1c159ba1508324c1f1f63)